packages/vite/src/node/__tests__/plugins/importGlob/parse.test.ts

```diff
@@ -272,10 +272,25 @@ describe('parse negatives', async () => {
     expect(
       await runError('import.meta.glob(`hi ${hey}`)')
     ).toMatchInlineSnapshot(
-      '[Error: Invalid glob import syntax: Could only use literals]'
+      '[Error: Invalid glob import syntax: Expected glob to be a string, but got contains expressions]'
     )
   })
 
+  it('template without expressions', async () => {
+    expect(await run('import.meta.glob(`/**/*.page.client.*([a-zA-Z0-9])`)'))
+      .toMatchInlineSnapshot(`
+      [
+        {
+          "globs": [
+            "/**/*.page.client.*([a-zA-Z0-9])",
+          ],
+          "options": {},
+          "start": 0,
+        },
+      ]
+    `)
+  })
```

Comment:
```js
import.meta.glob(`/\u0068\u0065\u006c\u006c\u006f`)
```
Could you add this test case too, in case?

Reply:
Of course and this test case is working!

Reply:
Thank you!